week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,20 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(grade) {
+  if (grade >= 80) {
+    return "A";
+  }
+  if (grade < 80 && grade > 60) {
```

Comment:
This is similar to the last comment. We know `grade < 80` at this point so you don't need this check. Same for `grade <= 60` below.